lib/src/utils.dart

```diff
@@ -245,6 +245,54 @@ String getLineEnding(String yaml) {
   return windowsNewlines > unixNewlines ? '\r\n' : '\n';
 }
 
+
+int _sizeOfScalar(dynamic value) => value == null ? 4 : '$value'.length;
```

Comment:
```suggestion
int _sizeOfScalar(YamlScalar? value) => value == null ? 4 : '$value'.length;
```
Shouldn't this be a nullable, or a nullable `YamlNode`?

Also is this true for all scalars? Just curious..